terraform/database.tf

```diff
@@ -0,0 +1,130 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+variable "database_password" {}
+
+resource "aws_vpc" "data_refinery_vpc" {
+  cidr_block = "10.0.0.0/16"
+  enable_dns_support = true
+  enable_dns_hostnames = true
+
+  tags {
+    Name = "data-refinery"
+  }
+}
+
+resource "aws_subnet" "data_refinery_1a" {
+  availability_zone = "us-east-1a"
+  cidr_block = "10.0.0.0/17"
+  vpc_id = "${aws_vpc.data_refinery_vpc.id}"
+  map_public_ip_on_launch = true
+
+  tags {
+    Name = "data-refinery-1a"
+  }
+}
+
+resource "aws_subnet" "data_refinery_1b" {
+  availability_zone = "us-east-1b"
```

Comment:
No. We don't actually have to worry about being in multiple availability zones for the workers because they don't have to always be available. In fact because we're planning on using spot instances they really won't be. Once there is a webserver then this will be a concern. (If this question was about why we have 2, that's to make sure that they can coordinate with each other to not repeat work etc.)